etc/config/typescript.amazon.properties

```diff
@@ -0,0 +1,18 @@
+# Default settings for TypeScript Compiler
+compilers=&tsc
+compilerType=typescript
+versionFlag=--version
+
+group.tsc.compilers=tsc_jit_gc:tsc_jit_nogc
+compiler.tsc_jit_gc.exe=/opt/compiler-explorer/tsc/bin/tsc
```

Comment:
This needs to be updated to point at the location (the updated one). I'd probably also rename it to `tsc_x` where `x` is the version (like `0.0.20`)

The path it ended up with is ```/opt/compiler-explorer/tsc-0.0-pre-alpha20/bin/tsc```

Reply:
(I may update this myself if there's no other issues, just to get this live :))